core/src/main/java/org/mule/runtime/core/internal/streaming/bytes/factory/NullCursorStreamProviderFactory.java

```diff
@@ -38,7 +39,7 @@ public NullCursorStreamProviderFactory(ByteBufferManager bufferManager, Streamin
    */
   @Override
   protected Object resolve(InputStream inputStream, EventContext creatorRootEventContext) {
-    streamingManager.manage(inputStream, creatorRootEventContext);
+    streamingManager.manage((Closeable) inputStream, creatorRootEventContext);
```

Comment:
could `inputStream` not be a closeable here?

Reply:
nope. InputStream class implements Closeable